src/app/auth/services/auth.service.ts

```diff
@@ -217,7 +222,7 @@ export const changePassword = async (newPassword: string, currentPassword: strin
   const privateKey = Buffer.from(user.privateKey, 'base64').toString();
   const privateKeyEncrypted = aes.encrypt(privateKey, newPassword, getAesInitFromEnv());
 
-  const usersClient = SdkFactory.getInstance().createUsersClient();
+  const usersClient = SdkFactory.getNewApiInstance().createNewUsersClient();
```

Comment:
Does the new key now have to be sent to the server-wip and not to the server? Wouldn't this break any functionality?